cmd/aergocli/cmd/contract.go

```diff
@@ -128,6 +129,15 @@ func init() {
 	rootCmd.AddCommand(contractCmd)
 }
 
+func isHexString(s string) bool {
+	for _, c := range s {
```

Comment:
It can be more safe by checking the length of string is even

Reply:
But the output of `aergoluac` can also be even...

Reply:
Oh, do you mean only inside the `isHexString()` function? Makes sense